man/resolved.conf.xml

```diff
@@ -317,6 +317,21 @@
         <xi:include href="version-info.xml" xpointer="v232"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RefuseRecordTypes=</varname></term>
+	<listitem><para>Takes a list of DNS record types separated by space.
+	Specified record types in each query will be refused. This option can be specified multiple times.
+	If an empty string is specified, then all previous assignments are cleared.</para>
```

Comment:
please follow our usual identation, as in the rest of the file.

Reply:
replaced tabs with spaces (vim can be annoying) :-(